### PR TITLE
social follows v2 qa bugs

### PR DESCRIPTION
**Pull request by kaushalmeena**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -37,16 +37,6 @@ body {
   display: none; /* Chrome/Safari/Webkit */
 }
 
-.glass-effect-button {
```
Comment (aadilhasan) [inline src/App.css:40]: are we not using this anywhere ?

File: index.html [hunk 2]
```diff
@@ -22,7 +22,7 @@
       <!-- End Google Tag Manager -->
 </head>
 
-<body>
+<body class="no-scrollbar">
   <div id="root"></div>
```
Comment (aadilhasan) [inline index.html:26]: why do we need this class?

**Review by aadilhasan — APPROVED** [review]

File: src/App.css (hunk 1 above)
Comment (kaushalmeena) [reply src/App.css:40]: yes I checked, we are not using anywhere

File: index.html (hunk 2 above)
Comment (kaushalmeena) [reply index.html:26]: the scrollbar track was taking width on the right side, resulting in slightly more gap on the right side, that's why hidden the scroll bar

**Pull request merged**